### DISCUSSION TITLE
Connecting Trimble R8S rover (with base station) to android device with qfield

### DISCUSSION THREAD

**Discussion by dukytony** [opengisch/QField#2495] [Q&A]
I'm would like to know if it's possible to use qfield with a trimble R8S rover in rtk mode (connected to a base module on a known position with radio) and how would that work?

So here is the plan : 

- connect the mobile rover to the base station with the trimble controller using bluetooth connection and the radio signal.
- disconnect the bluetooth connection from the trimble controller to the trimble mobile rover.
- connect the android device directly to the trimble rover with bluetooth? Use lefebure ntrip client in mock location?
- setup qfield to use mock location app?

Please let me know if that would work?

thanks a lot!

**Comment by geomondego** [comment]
@dukytony In the past, i used a Trimble R1 through the Trimble Mobile Manager app which is then used as the Mook location. Procedure shouldn't be much different with the R8S.

Regards
Nelson

**Reply by dukytony** [reply]
So you're not using the lefebure app?

**Reply by geomondego** [reply]
Just the Trimble Mobile Manager https://play.google.com/store/apps/details?id=com.trimble.trimblemobilemanager&hl=pt_PT&gl=US
This app allow you to use NTRIP connection.

Nelson

**Comment by om1s186** [comment]
You should use lefebure, TMM is o.k. but i couldn't manage to get orthometric height in Qfield with R8s. Lefebure has option which let you choose which height will use for mock location orthometric or ellipsoidal.

**Reply by dukytony** [reply]
Okay and this works with another module as base station (post-processed known location point)? I mean not using NTRIP corrections?

**Reply by om1s186** [reply]
What are you using as base station? For example R12 can enable NTRIP caster for NTRIP stream that can be used by Lefebure.

**Reply by dukytony** [reply]
I'm usually using R2 as rover and R8S as base station on a known point. What I'm wondering is : I will need to use the bluetooth connexion of the controller TDC600 to launch the base station through trimble access. Then I ll need to use trimble access to connect the rover to the base station by radio (tdl450). Then what do i do? should I install lefebure and qfield directly on the controller? then launch lefebure and it will receive nmea stream from the rover through bluetooth? Is there a way to be able to do this but from another android device (smartphone or tablet?). I dont think the R8S as a built in ntrip caster.
